script.js

```diff
@@ -69,7 +70,23 @@ function showTitle() {
   showTitleWrapper.appendChild(showTitle);
 }
 
+/***** TV-MAZE LINK *****/
+
+function createTvMazeLink() {
+  const tvMazeWrapper = document.createElement("div");
+  tvMazeWrapper.classList.add("tv-maze-wrapper");
+  body.appendChild(tvMazeWrapper);
+  const tvMazeLink = document.createElement("a");
+  tvMazeLink.setAttribute("target", "_blank");
+  tvMazeLink.setAttribute("rel", "noreferrer noopener");
+  tvMazeLink.classList.add("tv-maze-link");
+  tvMazeLink.innerHTML = "SOURCED FROM TV-MAZE.COM";
+  tvMazeLink.href = "https://www.tvmaze.com";
+  tvMazeWrapper.appendChild(tvMazeLink);
+}
+
 /***** FORMAT SHOW DATA *****/
+
 function formatShowData(seasonNum, episodeNum) {
```

Comment:
I really like the fact that you created a separate function to format the display of episodes' numbers.